README.rst

```diff
@@ -0,0 +1,85 @@
+======
+mtools
+======
+
+|PyPI version| |Build Status| |Python 27|
+
+**mtools** is a collection of helper scripts to parse, filter, and visualize
+MongoDB log files (``mongod``, ``mongos``). mtools also includes ``mlaunch``, a
+utility to quickly set up complex MongoDB test environments on a local machine.
+
+.. figure:: ./mtools.png
+   :alt: mtools box
```

Comment:
GitHub is rendering the alt text after the image: https://github.com/kallimachos/mtools/blob/4fa811efd1f28a46d823e9df19fb62e548cbf32d/README.rst

Reply:
Hrm. Originally there was a `<p>` with that text in it that I removed in an additional commit. I think you might be looking at an outdated version. I'll push the other changes and then check.

Reply:
I don't see an extra text after the image in the latest render. Could you please confirm?